bigbluebutton-html5/imports/ui/components/chat/component.jsx

```diff
@@ -91,7 +91,7 @@ const Chat = (props) => {
                 hideLabel
                 onClick={() => {
                   actions.handleClosePrivateChat(chatID);
-                  Session.set('idChatOpen', 'public');
+                  Session.set('idChatOpen', '');
```

Comment:
When you did private chat did you make this change and now you are reverting?

Reply:
yes sir

Reply:
Then this should not appear in the final change, if you reverted all the old ones. I should see only the other file change. Looks like you added this new now. Without this change are things not working?

Reply:
this change was done long ago  pushed to chatupload when private chat is enabled  , thats why its showing as change sir